src/modules/dart.rs

```diff
@@ -32,8 +33,8 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
             })
             .map(|variable| match variable {
                 "version" => {
-                    let dart_version =
-                        parse_dart_version(&context.exec_cmd("dart", &["--version"])?.stderr)?;
+                    let command = context.exec_cmd("dart", &["--version"])?;
+                    let dart_version = parse_dart_version(&get_command_string_output(command))?;
```

Comment:
Can you add a test with stdout version output? You can use `cmd` to mock the command dynamically ([example](https://github.com/starship/starship/blob/c1f2d345aac0b0241ea1b6d99977fea20fa3f5bb/src/modules/ocaml.rs#L339-L346)).

Reply:
Ok i will do it as soon as possible